doc/api/cli.md

```diff
@@ -1663,6 +1652,17 @@ changes:
 
 Disable the experimental [`node:sqlite`][] module.
 
+### `--no-experimental-strip-types`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.1 - Active development
```

Comment:
I think
```suggestion
> Stability: 1.2 - Release candidate
```

Reply:
Id wait a bit to move it to 1.2 there is still some work to do